reporter-config.json

```
@@ -0,0 +1,6 @@
+{
+  "reporterEnabled": "spec, json",
```

is there not a dot missing here? Just wondering.

```suggestion
  "reporterEnabled": ".spec, .json",
```

---

No those are names of reporters not file extensions